agent/installer/bundle_builder/build-bundle.sh

```diff
@@ -36,6 +36,6 @@ mkdir .imgpkg
 kbld --imgpkg-lock-output .imgpkg/images.yml
 
 echo Creating bundle tar
-tar -cvf /bundle/bundle.tar *
+tar -cvf /bundle/bundle.tar * .imgpkg
```

Comment:
Nit: Will it be more future-proof if
https://askubuntu.com/questions/829796/how-should-i-glob-for-all-hidden-files